src/main/java/net/minecraftforge/client/model/generators/BlockStateProvider.java

```diff
@@ -188,6 +188,10 @@ public void simpleBlockItem(Block block, ModelFile model) {
         itemModels().getBuilder(key(block).getPath()).parent(model);
     }
 
+    public void simpleBlockWithItem(Block block, ModelFile model) {
+        simpleBlock(block, model);
+        simpleBlockItem(block, model);
+    }
     public void simpleBlock(Block block, ConfiguredModel... models) {
```

Comment:
Please introduce a blank line between these two methods.
```suggestion
    }

    public void simpleBlock(Block block, ConfiguredModel... models) {
```